HelpSource/Classes/Ringz.schelp

```diff
@@ -6,9 +6,8 @@ categories::  UGens>Filters>Linear
 
 Description::
 
-This is the same as  link::Classes/Resonz:: , except that instead of a
-resonance parameter, the bandwidth is specified in a 60dB ring decay
-time. One Ringz is equivalent to one component of the
+This is the same as  link::Classes/Resonz:: , except that it is constant skirt gain filter, meaning that the peak gain depends on the value of Q. Also, instead of the
```

Comment:
whereas Resonz is a ___ ?

its great that you are adding more detail, but its leaving an unanswered question. if Resonz and Ringz are fundamentally different, then how are they different ?